### DISCUSSION TITLE
Add bun as an optional package manager.

### DISCUSSION THREAD

**Discussion by Stranger1586** [Idered/iridium#26] [Ideas]
Can [bun-js ](https://github.com/oven-sh/bun)(a JavaScript run time environment)  be included as an optional package manager along with npm/yarn?
I understand that bun is relatively immature but I do believe that some people might be interested in using it.

**Comment by Idered** [comment]
@Stranger1586 sounds good, I'll give it a try

**Comment by Idered** [comment]
@Stranger1586 Bun support landed in v1.6.0
